app/controllers/exams_controller.rb

```diff
@@ -0,0 +1,154 @@
+class ExamsController < ApplicationController
+  include ExamsHelper
+  before_action :find_exam, except: %i(create)
+
+  def show
+    @questions = @exam.questions
+    return unless @exam.start?
+
+    @exam.update status: 1
+    @exam.update endtime: Time.zone.now + @exam.subject.duration.minutes
```

Comment:
```suggestion
    @exam.update endtime: Time.zone.now + @exam.subject.duration.minutes, status: :pending 
```
bỏ dòng trên
k dc gọi 1,2 
gọi giá trị có nghĩa nhé